tests/plugin.test.ts

```diff
@@ -8,41 +8,42 @@ window.cordova = {
 
 describe("Purchases", () => {
   it("configure fires PurchasesPlugin with the correct arguments", () => {
-    Purchases.configure("api_key", "app_user_id");
-
-    expect(execFn).toHaveBeenCalledWith(
-      expect.any(Function),
-      null,
-      "PurchasesPlugin",
-      "configure",
-      ["api_key", "app_user_id", false, undefined]
-    );
+    Purchases.configure("api_key", () => {
+      expect(execFn).toHaveBeenCalledWith(
```

Comment:
This might be a newbie question, but this might not be called if the `configure` callback is not called right? Maybe we should add an assertion to make sure the callback gets called? Same for the next tests.

Reply:
great observation, I'll add something